src/screens/Home/HomeUtils.res

```diff
@@ -215,7 +215,7 @@ module ControlCenter = {
           />
           <img
             alt="connector-list"
-            className="inline-block absolute right-5 bottom-5  lg:block"
+            className="opacity-10 sm:opacity-100 inline-block absolute right-5 bottom-5  lg:block"
```

Comment:
hey I have addressed this issue in another PR you raised. You can remove it from this PR.

Reply:
Done!